app/controllers/api/custom_buttons_controller.rb

```diff
@@ -44,12 +42,8 @@ def build_custom_button_options
 
     def create_or_update_resource_action(data)
       return nil if data.empty?
-      id = data["id"]
-      if id.present?
-        resource_action = ResourceAction.find(id)
-        data.except!("id")
-        resource_action.update!(data.deep_symbolize_keys)
-        resource_action
+      if data["id"].present?
+        ResourceAction.find(id).tap { |ra| ra.update!(data.except!("id").deep_symbolize_keys) }
```

Comment:
is this right ?  doing find(id) but id is no longer defined.

Reply:
I totally did not see this

Reply:
AB with 🦅 👀